SpatialGDK/Documentation/index.md

```diff
@@ -1,58 +1,58 @@
 ![GDK for Unreal Documentation]({{assetRoot}}assets/spatialos-gdkforunreal-header.png)
 
-<%(Callout  message="The SpatialOS GDK for Unreal is in alpha. It is ready to use for development of games using a single server, or multiserver offloading functionality. We do not recommend development using multiserver zoning functionalty. For more information on multiserver zoning availability, see the [development roadmap](https://github.com/spatialos/UnrealGDK/projects/1) and [Unreal features support]({{urlRoot}}/unreal-features-support) page.")%>
+<%(Callout  message="The SpatialOS GDK for Unreal is in alpha. It can be used to develop games using a single server, and some multiserver functionality is also available, as described below.")%>
```

Comment:
Passive voice. Who can use it?

Should we call out the Roadmap? This was quite important to Ernie.

Reply:
We usually talk to developers as "you" or make it an instruction.